osmosis/assetlist.json

```diff
@@ -21345,6 +21345,31 @@
             "png":"https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COOK.png"
          }
       ]
-   }
+   },
+   {
+      "description": "ashSAIL",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1svj5kd8kzj7xxtrd6ftjk0856ffpyj4egz7f9pd9dge5wr4kwansmefq07/sail.ash",
+          "exponent": 0
+        },
+        {
+          "denom": "ashSAIL",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/osmo1svj5kd8kzj7xxtrd6ftjk0856ffpyj4egz7f9pd9dge5wr4kwansmefq07/sail.ash",
+      "name": "ashSAIL",
+      "display": "ashSAIL",
+      "symbol": "ashSAIL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/sail.ash.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/sail.ash.png"
```

Comment:
Are you able to provide an image without padding? They are uneven and looks kinda strange--likely unintentional.

Reply:
It looks like you re-uploaded the same image. It still has padding, and the file size is too large. Please upload an image < 250KB, (ideally < 200KB).